spec/factories/subjects.rb

```diff
@@ -16,7 +16,6 @@
     type {
```

Comment:
can we do any thing nice with

`SubjectCreatorService.new.execute` in the factory

given that the its no longer unknown...